www/docs/customization/homebrew_casks.md

```diff
@@ -242,8 +242,7 @@ homebrew_casks:
       post:
         # replace foo with the actual binary name
         install: |
-          if system_command("/usr/bin/xattr", args: ["-h"]).exit_status == 0
-            # replace 'foo' with the actual binary name
```

Comment:
i would leave this comment in one form or another...

Reply:
@caarlos0 
I think it's better to add comments on the YAML side rather than within the post install hook command, so I believe the description on L243 is sufficient.

Reply:
oh, sorry I'm blind, completely missed the other comment.